### DISCUSSION TITLE
Error : Email required

### DISCUSSION THREAD

**Discussion by Ronan-Lenor** [verschuur/generator-laravel-package-scaffolder#294] [Q&A] [answered]
Hello,
after executing `yo laravel-package-scaffolder`, i got this error :
```
Fetching GitHub info, please wait...
Error laravel-package-scaffolder

Email required
```
i don't understand from where it try to get those infos.
what step i'm missing?

**Comment by verschuur** [comment, marked as answer]
Did you configure your email address in the git config? What does it say when you run `git config --get user.email` in a terminal?

**Reply by Ronan-Lenor** [reply]
Hello,

i have installed GitHub desktop and connect throught it. it look like it has impacted my git CLI commands. Because `git config --get user.email`  and `yo laravel-package-scaffolder` work fine now.
(i'm not sure it come from GitHub desktop, but i suppose to. Coz i don't remember have doing anything else particular recently)

anyway, i didn't know that configure git was necessary to use that generator.
this would be a good idea to mention it in the documentation at the installation step.

**Reply by verschuur** [reply]
I was under the impression that when the GitHub fetch failed it would throw an error which would then be handled. Apparently it works differently. Anyway, thanks for the report, I'll have a look at it and update the readme.
